sample/src/main/java/co/nimblehq/sample/ui/surveylist/SurveyAdapter.kt

```diff
@@ -31,8 +33,15 @@ class SurveyAdapter : RecyclerView.Adapter<SurveyAdapter.SurveyViewHolder>() {
     }
 
     class SurveyViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
-        fun bind(value: String) {
-            (itemView as TextView).text = value
+        fun bind(value: SurveyModel) {
+            (itemView as TextView).text = value.surveyTitle
+            itemView.setOnClickListener {
+                val intent = Intent(itemView.context, SurveyDetailActivity::class.java).apply {
+                    putExtra("id", value.surveyId)
+                    putExtra("title", value.surveyTitle)
```

Comment:
- please create const for id and title
- we can pass the whole value object here into the detail screen

Reply:
I think later he will demonstrate the `GET survey/:survey_id`

Reply:
@sleepylee That's correct.
@luongvo Pass the whole object in this case is unnecessary, I think. 
Let me create the const for id and title